packages/block-editor/src/components/block-patterns/index.js

```diff
@@ -34,7 +34,7 @@ function BlockPattern( { pattern, onClick } ) {
 			tabIndex={ 0 }
 		>
 			<div className="block-editor-patterns__item-preview">
-				<BlockPreview blocks={ blocks } />
+				<BlockPreview blocks={ blocks } viewportWidth={ 800 } />
```

Comment:
I kept a smaller width here because most themes have a centered area around that size.